src/js/letters/containers/AddressSection.jsx

```diff
@@ -208,7 +208,11 @@ export class AddressSection extends React.Component {
       const city = address.city || '';
       const militaryStateCode = address.stateCode || '';
       cityStatePostal = `${city}, ${militaryStateCode} ${zipCode}`;
+    } else {
+      // Must be an international address, only show a city
+      cityStatePostal = `${address.city || ''}`;
```

Comment:
While outside of the scope of what this PR is trying to do, this logic looks like it could be cleaned up. Each condition basically checks for `address.city || ''`, so we could probably set `const city = address.city || ''` outside of the condition statements, as is done for `const zipCode`. And then this line could just be:
```cityStatePostal = `${city}`;
```
And then you can get rid of lines 204 and 208. I think it's a bit more concise that way.

Reply:
Thanks, good call!